tools/itsg33-to-gh-issues/script.py

```diff
@@ -1,31 +1,229 @@
+"""
+The purpose of this script is to take the CSV file with the ITSG-33 controls and create issues in github for each control.
+The script will create the issue title, body, and labels. The body will have the control definition, class, supplemental guidance,
+ references, general guide, suggested placeholder values, profile specific notes, suggested assignment, and support teams.
+"""
+
 import csv
 import requests
 import json
 import os
+import string
 
+"""
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+DEBUG = True/False to print debug info
+"""
 REPO = os.getenv("REPO")
 GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+CSV_FILE = "controls.csv"
+DEBUG = True
+FAMILY = 0
+CONTROL_ID = 1
+ENHANCEMENT = 2
+CONTROL_NAME = 3
+CONTROL_CLASS = 4
+CONTROL_DEFINITION = 5
+REFERENCES = 7
+SUPPLEMENTAL_GUIDANCE = 6
+IT_SECURITY_FUNCTION = 8
+IT_OPERATION_GROUP = 9
+IT_PROJECTS = 10
+PHYSICAL_SECURITY_GROUP = 11
+PERSONNEL_SECURITY_GROUP = 12
+LEARNING_CENTER = 13
+GENERAL_GUIDE = 14
+SUGGESTED_PRIORITY = 15
+SUGGESTED_PLACEHOLDER_VALUES = 16
+PROFILE_SPECIFIC_NOTES = 17
+LABELS = {
+        8: "IT Security Function", 
+        9: "IT Operation Group", 
+        10: "IT Projects", 
+        11: "Physical Security Group", 
+        12: "Personnel Security Group", 
+        13: "Learning Center"
+}
+
+
+def main():
+    
+    for row in get_controls():
+
+        issues_url = gh_issues_url()
+        headers = get_header()
+        issues_json = get_issues_json(row)
+        
+        response = post_request(issues_url, headers, issues_json)
+
+        if DEBUG:
+            print("Issues URL: {}".format(issues_url))
+            print("Issues JSON: {}".format(issues_json))
+            print("Response: {}".format(response.text))
+            print("Headers: {}".format(headers))
+
+
+def post_request(issues_url, headers, issues_json):
+    """
+    Post request to github api to create issue. If successful, print the control title. 
+    If not, print the control title and the response. Return the response.
+    """
+    response = requests.post(issues_url, headers=headers, json=issues_json)
+    if response.status_code == 201:
+        print("Created issue for control: {}".format(issues_json["title"]))
```

Comment:
For all your print statements, you could use `f` strings if you wanted.  They're a bit shorter, but what you have will also work:
```suggestion
        print(f"Created issue for control: {issues_json['title']}")
```

Reply:
Will do in the future!